dev-qt/qtwebengine/qtwebengine-5.7.0.ebuild

```diff
@@ -95,3 +98,9 @@ src_configure() {
 	)
 	qt5-build_src_configure
 }
+
+src_install() {
+	qt5-build_src_install
+
+	pax-mark m "${ED}"/usr/$(get_libdir)/qt5/libexec/QtWebEngineProcess
```

Comment:
About pax-marking unconditionally -- it looks like if pax-utils does not find a binary to paxmark executables, it'll simply log that the binary may not work on pax systems, so it doesn't require a system with elfix in order to function. On the other hand, the patch expects the system to have elfix, so I believe it should be masked behind the pax_kernel use flag. 

We could also get rid of the use flag and unconditionally depend on elfix for this package, but that doesn't really make sense to me.